base/linalg/diagonal.jl

```diff
@@ -233,3 +235,13 @@ function svdfact(D::Diagonal)
     U, s, V = svd(D)
     SVD(U, s, V')
 end
+
+#set related methods
```

Comment:
Comments in this file seem to be quite consistent in using `##` to delimit sections, with an initial upper case. Also, there's always a space after the `#`.